docs/meetings/2022/TechArea20220823.md

```diff
@@ -2,38 +2,34 @@
 
 -   **Coordinates:** Conference: +1-415-655-0002, PIN: 146 266 9392,
     <https://morgridge-org.zoom.us/j/91987518094> (password sent separately)
--   **Attending:** 
+-   **Attending:** BrianL, Mat, TimT, Carl, Marco Mambelli, Ziyang
 
 ## Announcements
 
 ### Triage Duty
 
 -   This week: Brian
 -   Next week: Carl
--   8 (+2) open FreshDesk tickets
--   0 (+0) open GGUS ticket
 
 ### Jira (as of Monday)
 
-| # of tickets | &Delta; | State             |
-|--------------|---------|-------------------|
-| 187          | -7      | Open              |
-| 38           | +6      | Selected for Dev  |
-| 35           | +5      | In Progress       |
-| 18           | +1      | Dev Complete      |
-| 3            | +0      | Ready for Testing |
-| 0            | +0      | Ready for Release |
+Lots of tickets should be backlogged.
```

Comment:
Could you revert the changes here? I still need to update these numbers and need to include them

Reply:
Done